### PR TITLE
Tile layer classes

### PR DESCRIPTION
**Pull request by tarleb**
Use classes to define some common tile layers. OSM and MapQuest are supported for now.

### CODE REVIEW

File: layers/tiles/MapQuestTileLayer.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+use dosamigos\leaflet\layers\TileLayer;
+
+/**
+ * An MapQuest tile layer
+ *
+ * @author Albert Krewinkel <albert+php@branchen.guru>
+ * @link http://developer.mapquest.com
+ * @package dosamigos\leaflet\layers\tiles
+ */
+class MapQuestTileLayer extends TileLayer
```
Comment (tonydspaniard) [inline layers/tiles/MapQuestTileLayer.php]: I think the name used is wrong. If you work with namespaces, MapQuest extending TileLayer is more than enough. As its within the tiles namespace.

File: layers/tiles/OsmTileLayer.php [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+use dosamigos\leaflet\layers\TileLayer;
+
+/**
+ * An Open Street Map tile layer
+ *
+ * @author Albert Krewinkel <albert+php@branchen.guru>
+ * @package dosamigos\leaflet\layers\tiles
+ */
+class OsmTileLayer extends TileLayer
```
Comment (tonydspaniard) [inline layers/tiles/OsmTileLayer.php]: Should be just Osm or OpenStreetMap, the suffix `Layer` is not necessary.

**Comment by tonydspaniard** [comment]
All good... Once the names are changed I will merge. Thanks for your work @tarleb

File: layers/tiles/MapQuestTileLayer.php (hunk 1 above)
Comment (tarleb) [inline layers/tiles/MapQuestTileLayer.php]: Naming things: One of the hardest things in CS. :smile:

**Comment by tonydspaniard** [comment]
Thanks!

**Pull request merged**